### PR TITLE
update running clair readme

### PR DESCRIPTION
**Pull request by ldelossa**
PR which syncs the "running-clair" readme with recent changes to local development and deployment options.

### CODE REVIEW

File: Documentation/running-clair.md [hunk 1]
```diff
@@ -83,6 +56,11 @@ $ docker run --net=host -d -p 6060-6061:6060-6061 -v $PWD/clair_config:/config q
 
 #### Source
 
+Clair currently uses glide as it's package management solution. The vendor folder is checked into this repository and running 'glide up' should not be necessary.
```
Comment (jzelinskie) [inline Documentation/running-clair.md]: sentence per line

File: Documentation/running-clair.md [hunk 2]
```diff
@@ -83,6 +56,11 @@ $ docker run --net=host -d -p 6060-6061:6060-6061 -v $PWD/clair_config:/config q
...
+Find more information on glide here: [Glide]
+
+[Glide]: https://github.com/Masterminds/glide
+
```
Comment (jzelinskie) [inline Documentation/running-clair.md:64]: How do you feel about removing the source section in general?
It seems semi-redundant with the local development doc now.

**Review by jzelinskie — CHANGES_REQUESTED** [review]

File: Documentation/running-clair.md (hunk 2 above)
Comment (ldelossa) [reply Documentation/running-clair.md:64]: @jzelinskie there was a bit of talk here: https://github.com/coreos/clair/issues/790 about having "general" running of clair information. IMO it does not hurt to have source build info if ppl want to just build the binary in their CI systems or other needs.

File: Documentation/running-clair.md (hunk 1 above)
Comment (ldelossa) [reply Documentation/running-clair.md]: fixed.

File: Documentation/running-clair.md (hunk 2 above)
Comment (ldelossa) [reply Documentation/running-clair.md:64]: also not sure where else we'd mention the required binaries in path?

Comment (ldelossa) [reply Documentation/running-clair.md:64]: ping @jzelinskie

Comment (jzelinskie) [reply Documentation/running-clair.md:64]: Forget it then, this is fine for now.

**Review by jzelinskie — APPROVED** [review]

**Pull request merged**